app/controllers/questions_controller.rb

```diff
@@ -1,2 +1,26 @@
 class QuestionsController < ApplicationController
+  before_action :set_quiz
+
+  def new
+    @question = @quiz.questions.build
+  end
+
+  def create
+    @question = @quiz.questions.build(question_params)
+    if @question.save
+      redirect_to quiz_path(@quiz)
+    else
+      render :new
+    end
+  end
+
+    private
+
+      def find_quiz
+        @quiz = Quiz.find(params[:quiz_id])
+      end
+
+      def question_params
+        params.require(:question).permit(:content, :correct_answer)
+      end
```

Comment:
Indentation does look a bit weird here but we can sort out on a PR after to add rubocop-govuk